Gemfile

```diff
@@ -2,52 +2,28 @@
 
 source 'https://rubygems.org'
 
-# Allow the rails version to come from an ENV setting so CI can test multiple versions.
-# See http://www.schneems.com/post/50991826838/testing-against-multiple-rails-versions/
-rails_version = ENV['RAILS_VERSION'] || '7.2.0'
-
-gem 'rails', rails_version.to_s
-
+gem 'appraisal'
 gem 'responders'
+gem 'sqlite3'
 
-case rails_version.split('.').first
-when '5'
-  gem 'sqlite3', '~> 1.3.6'
-when  '6', '7'
-  gem 'sqlite3', '~> 1.4.1'
-end
-
-case RUBY_VERSION.split('.').first
-when '3'
-  gem 'net-smtp', require: false
-end
+gem 'net-smtp', require: false if RUBY_VERSION.split('.').first == '3'
 
 gem 'rswag-api', path: './rswag-api'
+gem 'rswag-specs', path: './rswag-specs'
 gem 'rswag-ui', path: './rswag-ui'
 
-group :development, :test do
-  gem 'rswag-specs', path: './rswag-specs'
-end
-
-group :test do
-  gem 'capybara'
-  gem 'climate_control'
-  gem 'geckodriver-helper'
-  gem 'generator_spec'
-  gem 'rspec-rails'
-  gem 'selenium-webdriver'
-  gem 'test-unit'
-  gem 'simplecov', '=0.21.2'
-end
-
-group :development do
-  gem 'rubocop'
-end
-
-group :assets do
-  gem 'mini_racer'
```

Comment:
I've removed the `mini_racer` gem as it was causing intermittent failures and it seems like the project no longer requires it as a dependency.